<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База данных</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer_1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="updateButton">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>U</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectionButton">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="addresseeTab">
         <attribute name="title">
          <string>Адрес</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTableView" name="addresseeView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="itemTab">
         <attribute name="title">
          <string>Предмет</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableView" name="itemView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="attachmentTab">
         <attribute name="title">
          <string>Вложение</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTableView" name="attachmentView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mailTab">
         <attribute name="title">
          <string>Почтовое отправление</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableView" name="mailView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="typeTab">
         <attribute name="title">
          <string>Тип отправления</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTableView" name="typeView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="officeTab">
         <attribute name="title">
          <string>Отделение связи</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QTableView" name="officeView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="2">
         <widget class="QPushButton" name="anyButton">
          <property name="text">
           <string>ANY</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="caseButton">
          <property name="text">
           <string>CASE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <item>
           <property name="text">
            <string>ИД_Отправление</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Индекс</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ценность</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Вес</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Общая_Ценность</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Стоимость</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="havingButton">
          <property name="text">
           <string>HAVING</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="viewButton">
          <property name="text">
           <string>VIEW</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
